lib/generators/rspec/install/templates/spec/rails_helper.rb

```diff
@@ -28,8 +28,7 @@
 begin
   ActiveRecord::Migration.maintain_test_schema!
 rescue ActiveRecord::PendingMigrationError => e
-  puts e.to_s.strip
-  exit 1
+  abort "#{e}".strip
```

Comment:
There is a difference, though. `abort` outputs to STDERR, while `puts` to STDOUT.
I tend to think that STDERR is more appropriate here.

Cosmetic: I'd prefer to keep `e.to_s`. Is there a reason to use string interpolation?

Reply:
I agree that STDERR seems to be more appropriate. No reason to use string interpolation. I'll go ahead and update the PR. Thank you very much.